src/main/js/store/actions/globalConfiguration.js

```diff
@@ -268,8 +269,8 @@ export function getConfig(descriptorName) {
             response.json()
                 .then((responseData) => {
                     if (response.ok) {
-                        if (responseData.length > 0) {
-                            dispatch(configFetched(responseData[0]));
+                        if (responseData.fieldModels && responseData.fieldModels.length > 0) {
```

Comment:
Do providers use these actions? If not, then we should use `length == 1` here I think

Reply:
providers do use these actions but not this function specifically.  I will add the change and test.